### ISSUE TITLE
Missing warning from log file

### ISSUE THREAD

**Issue by ehogan** [PCMDI/cmor#73]
I made some changes so that the example worked. My log file contains:

```
% more cmor.log
! ------
! CMOR is now closed.
! ------
! During execution we encountered:
!   1 Warning(s)
!   0 Error(s)
! ------
! Please review them.
! ------
! 
%
```

There is a warning that hasn't made it to the log file. I think it may [this warning](https://github.com/PCMDI/cmor/blob/master/Src/cmor.c#L785).

**Comment by dnadeau4** [comment]
What warning do you get if you do not setup a log file?

**Comment by ehogan** [comment]
Using the latest version of CMOR and specifying a log in the example:
```
% ./example.py
% more cmor.log

! ------
! All files were closed successfully. 
! ------
! 
% ./example.py 
% more cmor.log
! ------
! CMOR is now closed.
! ------
! During execution we encountered:
!   1 Warning(s)
!   0 Error(s)
! ------
! Please review them.
! ------
! 
% emacs example.py  # comment out logfile
% ./example.py 

! ------
! All files were closed successfully. 
! ------
! % ./example.py 

! ------
! All files were closed successfully. 
! ------
! %
```
I don't get a warning if I don't specify a log file, even if I execute the example more than once. I'm not sure where the warning is going?

**Comment by dnadeau4** [comment]
I have not work on the log file lately.  I would like to see the "All files were closed successfully." message in the log as well.  I might open/close the file for each message.   The "1 warning(s)" message without the warning is worrisome.  I don't know why this happened.

**Comment by durack1** [comment]
@dnadeau4 is this a candidate for `3.2.0` or is it done?

**Comment by dnadeau4** [comment]
I don't see this issue anymore.  @ehogan?

**Comment by ehogan** [comment]
Using CMOR 3.2.2 I still get:
```
% ./example.py 
% more cmor.log 

! ------
! All files were closed successfully. 
! ------
! 
% ./example.py 
% more cmor.log 
! ------
! CMOR is now closed.
! ------
! During execution we encountered:
!   1 Warning(s)
!   0 Error(s)
! ------
! Please review them.
! ------
! 
```

**Comment by ehogan** [comment]
P.S. `example.py` contains the code in the `try` loop here: https://github.com/PCMDI/cmor/blob/master/Test/test_python_CMIP6_experimentID.py#L59 but with `logfile='cmor.log'` in the call to `cmor.setup`.

**Comment by durack1** [comment]
@dnadeau4 move to `3.2.3`?

**Comment by dnadeau4** [comment]
I fixed it, it was the `rename` warning that was not print.  

```
C Traceback:
In function: 

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Warning: Logfile mytest.log already existed.
! Renamed to: mytest.log_2017-04-12T17:12:31
!
!!!!!!!!!!!!!!!!!!!!!!!!!


C Traceback:
In function: _CV_checkExperiment
! called from: cmor_setGblAttr
! 

!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Warning: Your input attribute "sub_experiment_id" with value 
! "abcd" will be replaced with value "none"
! as defined for experiment_id "piControl-withism".
! 
!  See Control Vocabulary JSON file.(Tables/CMIP6_CV.json)
! 
!
!!!!!!!!!!!!!!!!!!!!!!!!!

! ------
! CMOR is now closed.
! ------
! During execution we encountered:
!   2 Warning(s)
!   0 Error(s)
! ------
! Please review them.
! ------
! 
```

**Comment by dnadeau4** [comment]
Fixed in CMOR 3.2.3

**Comment by ehogan** [comment]
I can confirm this works using CMOR 3.2.3; thanks @dnadeau4! :)
